keyboards/eek/rules.mk

```diff
@@ -0,0 +1,27 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = qmk-dfu
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = yes        # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+
+#RGB_MATRIX_ENABLE = WS2812 # Enable for pretty RGB matrix effects
```

Comment:
```suggestion
RGB_MATRIX_DRIVER = WS2812
```

Reply:
I had to make some corrections to this RGB matrix. I realized that I pulled this from a branch that had some issues with the pins and led_config_t struct. I corrected those and updated the eek.c, config.h, rules.mk (both in main folder and ledtest subfolder), and the ledtest keymap comments.

I added in the  ``RGB_MATRIX_DRIVER = WS2812`` but, I'm hesitant to remove the ``#RGB_MATRIX_ENABLE = WS2812 # Enable for pretty RGB matrix effects`` because if someone wants to enable it and make changes in their own personal keymap then they will need to copy and paste that file and uncomment that line for the RGB matrix to work, right? At least,  I'm not sure how to make it work without that line in the rules.mk.

Reply:
Stuff from higher level rules.mk files is inherited by lower level ones, so users can simply add a new rules.mk in their keymap directory that just says `RGB_MATRIX_ENABLE = yes`, without having to copy the keyboard-level file or care which specific driver the board is using.

Reply:
Also, part of the point of having the separate driver setting, is to that the "enable" feature can be yes/no, so you don't actually have to know what driver it's using.  

This is ESPECIALLY useful for community layouts, as you can have several different boards that use different drivers, and not have to worry about it.